content/docs/user-guide/packaging/project-export/project-export-cli.md

```diff
@@ -131,13 +170,60 @@ An example usage of the entire `export-project` command, including this script,
 ```
 Where `O3DE_ENGINE_PATH`, `O3DE_PROJECT_PATH` and `OUTPUT_PATH` are environment variables. This single invocation should be all that is needed to fully export MultiplayerSample into a release directory ready for distribution.
 
-{{< important >}}
---no-monolithic-build and --config profile is required when building from an O3DE installer or SDK. 
-See [Installed Engine Fails to Export Monolithic Release.](https://github.com/o3de/o3de/issues/17203)
-{{< /important >}}
-
 For more information on how to export the MultiplayerSample project using the CLI, please see [these instructions](https://github.com/o3de/o3de-multiplayersample/blob/f00b3035285b695b2dbd1b1e59912973f4e1a32f/Documentation/O3DEMPSProjectExportTesting.md).
 
+## Export Script Configurations
+
+The standard export script provides functionality for managing your export configurations. Configuration is set either globally (for all projects), or on a per-project basis. See the help section of the `export-project-configure` command for more details:
+```cmd
+<O3DE_ENGINE_PATH>\scripts\o3de.bat export-project-configure -h
+```
+
+You can see the current list of configured settings using:
+```cmd
+<O3DE_ENGINE_PATH>\scripts\o3de.bat export-project-configure -l
+
+# To see the configured values globally across O3DE
+<O3DE_ENGINE_PATH>\scripts\o3de.bat export-project-configure --global -l
+
+# To see the configured values for a specific project (using the Project name registered with O3DE)
+<O3DE_ENGINE_PATH>\scripts\o3de.bat export-project-configure -p ProjectName -l
+```
```

Comment:
Separate each snippet to its own section, and place the description outside of it (ie remove from the #comment)

Reply:
You can skip the linux example for now